File: GeometriaJava/src/geometria/Cuadrado.java
package geometria;

public class Cuadrado extends Geometria2D {
    private int coordenadaX;
    private int coordenadaY;
    private int lado;
    
    public Cuadrado(int x, int y, int lado) {
        this.coordenadaX = x;
        this.coordenadaY = y;
        this.lado = lado;
    }
    
    @Override
    public double calcularArea() {
        return lado * lado;
    }
    
    @Override
    public double calcularPerimetro() {
        return 4 * lado;
    }
    
    public int getCoordenadaX() {
        return coordenadaX;
    }
    
    public int getCoordenadaY() {
        return coordenadaY;
    }
    
    public int getLado() {
        return lado;
    }
}

File: GeometriaPython/geometria/cuadrado.py
from geometria.geometria2d import Geometria2D

class Cuadrado(Geometria2D):
    def __init__(self, x, y, lado):
        self.coordenadaX = x
        self.coordenadaY = y
        self.lado = lado

    def calcular_area(self):
        return self.lado * self.lado

    def calcular_perimetro(self):
        return 4 * self.lado

    def get_coordenadaX(self):
        return self.coordenadaX

    def get_coordenadaY(self):
        return self.coordenadaY

    def get_lado(self):
        return self.lado
